src/main/java/io/orangebeard/listener/helper/ToolchainRunningContext.java

```diff
@@ -50,6 +53,15 @@ public UUID getSuiteId(String fullSuiteName) {
         return null;
     }
 
+    public String getSuitePath(UUID suiteUUID) {
+        for (Map.Entry<String, Suite> entry : suites.entrySet()) {
+            if (entry.getValue().getUuid().equals(suiteUUID)) {
+                return entry.getKey();
+            }
+        }
+        return null;
+
+    }
```

Comment:
This implementation does not make any sense. Why have a map if you're going to loop through it? I think it's better if you make another map to be able to provide the info fast.